### ISSUE TITLE
darknet.exe has stop working during training  pthread_join

### ISSUE THREAD

**Issue by Zhaowei-Yu** [AlexeyAB/darknet#2780]
Hi,@AlexeyAB
I've set the darknet platform as your guide on my (windows 7+ core 6700k +rtx 2070 +8G RAM +VS2015 + OPENCV2.4.9+cuda 10.1) computer, the compile process is successful, but when I start training, it displayed "darknet.exe has stopped working". Always stops working at a breakpoint shown in the pic. The win32 console stopped like this :
![image](https://user-images.githubusercontent.com/48342561/55270336-9784ff00-52d8-11e9-88f1-06d97068c067.png)
Critical error detected c0000374
0xC0000374: 堆已损坏
I have updated to the latest version of this repository, and there is no bad list.
Thank you very much

**Comment by AlexeyAB** [comment]
@Zhaowei-Yu Hi,

Can you show command that you use for Training, cfg file, obj.data file and the first several lines from train.txt ?

**Comment by Zhaowei-Yu** [comment]
> @Zhaowei-Yu Hi,
> 
> Can you show command that you use for Training, cfg file, obj.data file and the first several lines from train.txt ?

@AlexeyAB
1. darknet.exe detector train data/obj.data yolov3-my-obj.cfg yolov3-tiny.conv.15

2. 
`[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 10000
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear



[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
`
3. 
`classes= 1
train  = data/train.txt
valid  = data/valid.txt
names = data/obj.names
backup = backup/
`
4.
`data/obj/test4_00000000.jpg
data/obj/test4_00000001.jpg
data/obj/test4_00000002.jpg
data/obj/test4_00000003.jpg
data/obj/test4_00000004.jpg
data/obj/test4_00000005.jpg`

**Comment by AlexeyAB** [comment]
Try to copy OpenCV dll's to the folder where is darknet.exe file.

If it doesn't help, can you attach the samllest reduced dataset (several images, train.txt, obj.data, names, cfg-file) that allows to reproduce this error?

**Comment by Zhaowei-Yu** [comment]
After I copied OpenCV dll's to the folder where is darknet.exe file， it's working now. 
THANK YOU so much!!
